package.json

```diff
@@ -51,10 +52,12 @@
     "babel-plugin-react-transform": "^3.0.0",
     "babel-plugin-transform-react-constant-elements": "^6.5.0",
```

Comment:
These should be removable (pulled in by the react preset)

Reply:
I see it here
https://github.com/STRML/react-grid-layout/blob/master/webpack-examples.config.js#L21-L22